### PR TITLE
Feature/fp21 81 contact form

### PR DESCRIPTION
**Pull request by veronicaburduzhan**
# Description

## Frontend: 
Contact Us page with contact form component and message has been sent component, and page Inbox with incoming messages that will be visible only for an admin user. 
## Backend: 
Added routes for adding and retrieving data from the database (messages from users). Created migration for table user_enquiry where sent messages will be stored.
Screenshot 2022-08-15 at 14 12 10
Screenshot 2022-08-15 at 14 11 05
Screenshot 2022-08-15 at 14 10 44

# How to test?

1. Run yarn db:clean and yarn db:setup to update migration files.
2. Run both server and client and go to the contact us page by pressing the link on the navigation panel.
3. To access the admin page's inbox, go directly to http://localhost:3000/inbox-admin.
4. Run storybook to see message has been sent component.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  };
+
+  const handleSubmit = async (e) => {
+    if (
+      formValues.firstname === '' ||
+      formValues.email === '' ||
+      formValues.message === ''
+    ) {
+      e.preventDefault();
+      setFormError('Please fill in the required fields!');
```
Comment (somayehdvr) [inline packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js:44]: When it shows this message, the submit button goes down, which is not nice and compatible with the design. I suggest reconsideration.

**Review by somayehdvr — APPROVED** [review]
Good job!

File: packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js (hunk 1 above)
Comment (veronicaburduzhan) [reply packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js:44]: You're right. This bug happened after I clean up CSS files but didn't notice that button is moving. Will fix it in a bit:)

Comment (veronicaburduzhan) [reply packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js:44]: Done✨

**Review by Divyajg — COMMENTED** [review]
Everything looks good, but when the `Please fill in the required fields!` is showing, `setIsFilled has set to false` and after filling the fields `setIsFilled `is still `false`, so the submit button is not working and to make `setIsFilled true`, the page needs refresh.
The state `setIsFilled ` might go along with the change of input fields.

**Review by Divyajg — APPROVED** [review]
It is working as it should now.
Great work. 👍

**Pull request merged**